app/ui-sidenav/client/sidebarItem.js

```diff
@@ -169,26 +175,29 @@ Template.sidebarItem.onCreated(function() {
 			}
 		}
 
-		if (!currentData.lastMessage._id) {
+		if (currentData.lastMessage && !currentData.lastMessage._id) {
```

Comment:
@shubhsherl  Instead of checking for `currentData.lastMessage &&` in every if conditions, can we add one if block above?
```
if(!currentData.lastMessage){
    return;
}
```